dotnet-source-8.yaml

```diff
@@ -0,0 +1,210 @@
+package:
+  name: dotnet-source-8
```

Comment:
An option would be to leave the current file `dotnet-8.yaml` to use the bootstrap package.

Reply:
Ack.